pkg/kubectl/delete.go

```diff
@@ -203,8 +203,10 @@ func (reaper *ReplicationControllerReaper) Stop(namespace, name string, timeout
 			return err
 		}
 	}
-	falseVar := false
-	deleteOptions := &metav1.DeleteOptions{OrphanDependents: &falseVar}
+	// Using a background deletion policy because the replication controller
+	// has already been scaled down.
```

Comment:
scaling down doesn't mean all dependents are deleted, just pods. why not foreground here as well?

Reply:
actually, can we remove all the reapers and rely on server-side GC instead at this point?

Reply:
> can we remove all the reapers and rely on server-side GC instead

@liggitt I think the answer is "yes", but I would prefer this change to be a separate PR.
@caesarxuchao what do you think?

Reply:
> why not foreground here as well?
@liggitt I would also prefer to have foreground everywhere, but @caesarxuchao previously suggested to use background for all reapers, see https://github.com/kubernetes/kubernetes/pull/59851#discussion_r171072408